README.md

```diff
@@ -1800,25 +1804,24 @@ Typically, early to mid-stage companies grant stock options, and they may be
 - ☝️ This is partly because ISOs
   [can make it harder](http://thestartuplawblog.com/the-problem-with-immediately-exercisable-isos/)
   to meet the long-term capital gains holding period.
-  Many people expect early exercise together with an 83(b) election will help them hold the
-  stock longer, to qualify for long-term capital gains.
-  While this is true for NSOs, there is a murky part of the rules on ISOs that implies that
-  even with an 83(b) election, the capital gains holding period does not begin until the
-  shares actually vest.
-  So, if you want to immediately exercise an option and file a Section 83(b) election, and
-  you might have liquidity soon, it’s better if you can have it be an NSO.
+  Many people expect early exercise, together with an 83(b) election, will help them hold
+  the stock long enough to qualify for long-term capital gains.
+  While this is true for NSOs, a murky part of the rules on ISOs states that even with an
+  83(b) election, the capital gains holding period does not begin until the shares actually
+  vest. So if you want to immediately exercise an option and file a Section 83(b) election,
+  and you might have liquidity soon, it’s better if you can do so with NSOs.
 
 ### The AMT trap
 
 One scenario is so dangerous we give it its own section.
 
 ❗ If you have received an ISO, exercising it may unexpectedly trigger a big AMT bill—even
 before you actually make any money on a sale!
-If there is a large spread between strike price and 409A value, you are potentially on the
-hook for an enormous tax bill, even if you can’t sell the stock.
-This has pushed people into bankruptcy.
-It also caused Congress to grant a one-time forgiveness;
-the odds of that happening again are very low.
+If there is a large spread between the strike price and the value assigned by the 409A,
+you are potentially on the hook for an enormous tax bill, even if you can’t sell the
+stock. This has pushed people into bankruptcy.
+It also caused Congress to grant a one-time forgiveness—the odds of that happening again
+are very low.
 
 This is sometimes called the **AMT trap**. To make matters worse, you probably can’t sell
```

Comment:
Optional, not a copyedit, but we could make this a definition with rephrasing.

"🄳 The catastrophic scenario where exercising ISOs triggers a large AMT bill, with no ability to sell the stock to pay the taxes, is called the **AMT trap**." 

(This would be nice as it would add mouseovers elsewhere.)

Reply:
Fixed!